Anomaly/Patches/Misc/Mutants.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Patch>
+
+  <!-- ========== Shambler ========== -->
+  <Operation Class="PatchOperationReplace">
+    <xpath>Defs/MutantDef[defName="Shambler"]/tools</xpath>
+    <value>
+      <tools>
+        <li Class="CombatExtended.ToolCE">
+          <label>teeth</label>
+          <capacities>
+            <li>Bite</li>
+          </capacities>
+          <power>8.2</power>
+          <cooldownTime>2</cooldownTime>
+          <linkedBodyPartsGroup>Teeth</linkedBodyPartsGroup>
+          <chanceFactor>1</chanceFactor>
+          <soundMeleeHit>Pawn_Melee_HumanBite_Hit</soundMeleeHit>
+          <soundMeleeMiss>Pawn_Melee_HumanBite_Miss</soundMeleeMiss>
+          <armorPenetrationSharp>0.15</armorPenetrationSharp>
+          <armorPenetrationBlunt>1.0</armorPenetrationBlunt>
+        </li>
+        <li Class="CombatExtended.ToolCE">
+          <label>left hand</label>
+          <capacities>
+            <li>Scratch</li>
+          </capacities>
+          <power>7</power>
+          <cooldownTime>2</cooldownTime>
+          <linkedBodyPartsGroup>LeftHand</linkedBodyPartsGroup>
+          <chanceFactor>1.5</chanceFactor>
+          <armorPenetrationBlunt>1.2</armorPenetrationBlunt>
+          <armorPenetrationSharp>0.6</armorPenetrationSharp>
+          <alwaysTreatAsWeapon>true</alwaysTreatAsWeapon>
+        </li>
+        <li Class="CombatExtended.ToolCE">
+          <label>right hand</label>
+          <capacities>
+            <li>Scratch</li>
+          </capacities>
+          <power>7</power>
+          <cooldownTime>2</cooldownTime>
+          <linkedBodyPartsGroup>RightHand</linkedBodyPartsGroup>
+          <chanceFactor>1.5</chanceFactor>
+          <armorPenetrationBlunt>1.2</armorPenetrationBlunt>
+          <armorPenetrationSharp>0.6</armorPenetrationSharp>
+          <alwaysTreatAsWeapon>true</alwaysTreatAsWeapon>
+        </li>
+      </tools>
+    </value>
+  </Operation>
+
+  <!-- ========== Ghoul ========== -->
+  <Operation Class="PatchOperationReplace">
```

Comment:
The whole file is using the wrong indentation style.
I'd assume the ghouls have bioferrite claws and bump the sharp AP on their scratch attacks, probably increase the damage too. Something close to an unpowered power claw.
![image](https://github.com/CombatExtended-Continued/CombatExtended/assets/56392968/fa83bb6c-e7f3-43e7-b8c2-c46151509e0e)